Gemfile

```diff
@@ -46,10 +46,7 @@ end
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
   gem 'web-console', '>= 3.3.0'
-  gem 'listen', '~> 3.0.5'
```

Comment:
removing the gems listen, spring, and spring-watcher-listen, along with commenting out the line in development.rb, allowed me to avoid having to restart my server every time I wanted to change a model file.